frontend/src/components/CardCreator/CardCreator.tsx

```diff
@@ -1,131 +1,444 @@
-import React from 'react'
-import { Link } from 'react-router-dom'
-import "./CardCreator.css"
-import BlankCard from "../../Media/Cardimages/blank.png"
-import dark from "../../Media/Attributes/DARK.png"
-import light from "../../Media/Attributes/LIGHT.png"
-import divine from "../../Media/Attributes/DIVINE.png"
-import earth from "../../Media/Attributes/EARTH.png"
-import fire from "../../Media/Attributes/FIRE.png"
-import water from "../../Media/Attributes/WATER.png"
-import wind from "../../Media/Attributes/WIND.png"
-import spell from "../../Media/Attributes/SPELL.png"
-import trap from "../../Media/Attributes/TRAP.png"
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import "./CardCreator.css";
+
+import normalCard from "../../Media/Cardimages/common.png";
+import effectCard from "../../Media/Cardimages/effect.png";
+import fusionCard from "../../Media/Cardimages/fusion.png";
+import ritualCard from "../../Media/Cardimages/ritual.png";
+import synchroCard from "../../Media/Cardimages/synchro.png";
+import xyzCard from "../../Media/Cardimages/xyz.png"
+
+
+import dark from "../../Media/Attributes/DARK.png";
+import light from "../../Media/Attributes/LIGHT.png";
+import divine from "../../Media/Attributes/DIVINE.png";
+import earth from "../../Media/Attributes/EARTH.png";
+import fire from "../../Media/Attributes/FIRE.png";
+import water from "../../Media/Attributes/WATER.png";
+import wind from "../../Media/Attributes/WIND.png";
+
+
+import levelstar1 from "../../Media/Level/levelstar.png";
 
 function CardCreator() {
+  const [showedStars, setShowedStars] = useState(
+    <div className="star-container"></div>
+  );
+
+  const [showedRace, setShowedRace] = useState(<div></div>);
+  const [showedAttribute, setShowedAttribute] = useState(
+    <img src={dark} alt="" />
+  );
+  const [showedType, setShowedType] = useState(<div></div>);
+  const [showedCard, setShowedCard] = useState(<div className="card-container"><img className="card_img" src={effectCard} alt="" /></div>);
+  const [name, setName] = useState('');
+  const [description, setDescription] = useState('')
+  const [attack, setAttack] = useState("");
+  const [defence, setDefence] = useState("");
+
+
+  const changeDark = () => {
+    setShowedAttribute(<img src={dark} alt="" />);
+  };
+
+  const changeLight = () => {
+    setShowedAttribute(<img src={light} alt="" />);
+  };
+
+  const changeDivine = () => {
+    setShowedAttribute(<img src={divine} alt="" />);
+  };
+
+  const changeEarth = () => {
+    setShowedAttribute(<img src={earth} alt="" />);
+  };
+
+  const changeFire = () => {
+    setShowedAttribute(<img src={fire} alt="" />);
+  };
+
+  const changeWater = () => {
+    setShowedAttribute(<img src={water} alt="" />);
+  };
+
+  const changeWind = () => {
+    setShowedAttribute(<img src={wind} alt="" />);
+  };
+
+
+  const changeType = (types: string) => {
```

Comment:
I'm pretty sure this method actually only needs two lines. Add the card as parameter and remove the complexity.

Reply:
I tried it but only the type changes but the card does not. how can I add an Image as parameter?

Reply:
You could create a map with the type as key and the image as value, but I'm not sure if this would be much better. Just leave this one for now.